src/config/docker.md

```diff
@@ -0,0 +1,96 @@
+# Docker
+
+Docker is a platform of products to build isolated containers for programs using
+visualization.
+
+## Getting Started with Docker.
+
+After installing the 'docker' package (and optionally, the `docker-compose`
+package), enable the `docker` service:
+
+```
+# ln -s /etc/sv/docker /var/service
+```
+
+and add your user to the `docker` group:
+
+```
+# usermod -aG docker $USER
+```
+
+You will know you have properly installed docker if you can run it in the
```

Comment:
From here down to the `voidlinux` docker images could all be considered 'how to use docker' and I don't think that fits the criteria for inclusion in the docs. I think the 'for more information' link could stay.